pkg/remediator/remediator.go

```diff
@@ -92,16 +101,18 @@ var _ Interface = &Remediator{}
 // It is safe for decls to be modified after they have been passed into the
 // Remediator.
 func New(scope declared.Scope, syncName string, cfg *rest.Config, applier syncerreconcile.Applier, decls *declared.Resources, numWorkers int) (*Remediator, error) {
+	remediator := &Remediator{
+		fightErrs: map[core.ID]status.Error{},
```

Comment:
This kind of required initialization should be done with a New constructor, or modify the methods so they lazy init the map.

Reply:
DONE. Refactored to use a New constructor for the FightHandler initialization.